src/mbgl/programs/extrusion_texture_program.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <mbgl/programs/program.hpp>
+#include <mbgl/programs/attributes.hpp>
+#include <mbgl/programs/uniforms.hpp>
+#include <mbgl/shaders/extrusion_texture.hpp>
+#include <mbgl/style/layers/fill_extrusion_layer_properties.hpp>
+#include <mbgl/util/geometry.hpp>
+
+#include <string>
+
+namespace mbgl {
+
+struct ExtrusionTextureLayoutAttributes : gl::Attributes<
+attributes::a_pos>
```

Comment:
Nit: indent the template parameter lists by one level (⨉3 here, ⨉5 in `fill_extrusion_program.hpp`).